docs/table/access_table.rst

```diff
@@ -458,16 +455,14 @@ supported navigation keys are:
 pprint() method
 ^^^^^^^^^^^^^^^
 
-In order to fully control the print output use the Table
-:meth:`~astropy.table.Table.pprint` or Column
-:func:`~astropy.table.Column.pprint` methods. These have keyword
-arguments ``max_lines``, ``max_width``, ``show_name``, ``show_unit`` with
-meanings as shown below::
+In order to fully control the print output use the
+:meth:`astropy.table.Table.pprint` or :meth:`astropy.table.Column.pprint`
```

Comment:
This is a question of style preference, so it makes sense to adopt your suggestion if enough people support it. Unless someone more familiar with Sphinx knows a way to have these links show up as `Table.pprint()` etc because that would be clearly preferred.

Reply:
I am neutral on this, so I'll defer to @taldcroft or @mhvk as Table maintainers.

Reply:
Resolved with a 3rd solution. Thanks @eerovaher!